LibreNMS/Alert/Transport/Slack.php

```diff
@@ -85,14 +85,14 @@ public static function configTemplate(): array
                 ],
                 [
                     'title' => 'Icon',
-                    'name' => 'slack-author',
+                    'name' => 'slack-icon_emoji',
                     'descr' => 'Name of emoji for icon',
                     'type' => 'text',
                 ],
             ],
             'validation' => [
                 'slack-url' => 'required|url',
-                'slack-channel' => 'string',
+                'slack-author' => 'string',
```

Comment:
I think it was `slack-username` you wanted to to update here as `slack-channel` is a valid option

Reply:
You're correct, am changing things now. Sorry about that